h/static/scripts/controllers.coffee

```diff
@@ -48,7 +48,8 @@ class AppController
           annotationMapper.loadAnnotations data
         when 'delete'
           for annotation in data
-            $scope.$emit('annotationDeleted', annotation)
+            if a = threading.idTable[annotation.id]?.message
+              $scope.$emit('annotationDeleted', a)
```

Comment:
> Could you add a test to verify the fix and to prevent regressions

Umm.... I am trying to identify the relevant test suite, but until so far, I haven't found anything covering the part of AppController that is processing streamer messages.  (`streamer.onmessage`, `applyUpdates`, etc.)

Am I missing something here?